client/jetpack-cloud/sections/agency-dashboard/sites-overview/index.tsx

```diff
@@ -242,13 +270,24 @@ export default function SitesOverview() {
 						<SiteSurveyBanner isDashboardView />
 						<SiteWelcomeBanner isDashboardView />
 						<SiteDowntimeMonitoringUpgradeBanner />
-
 						{ data?.sites && <SiteAddLicenseNotification /> }
 						<SiteContentHeader
-							content={ renderIssueLicenseButton() }
+							content={
+								// don't render content on mobile, we don't need it
```

Comment:
Nitpick: When you read this comment, it looks like we don't display these buttons on mobile, which is incorrect.

Reply:
Alright, I've changed the wording, hopefully it's a little more clear.